docs/framework/interop/identifying-functions-in-dlls.md

```diff
@@ -21,23 +21,23 @@ The identity of a DLL function consists of the following elements:
   
 -   Name of the DLL file in which the implementation can be found  
   
- For example, specifying the **MessageBox** function in the User32.dll identifies the function (**MessageBox**) and its location (User32.dll, User32, or user32). The Microsoft Windows application programming interface (Win32 API) can contain two versions of each function that handles characters and strings: a 1-byte character ANSI version and a 2-byte character Unicode version. When unspecified, the character set, represented by the <xref:System.Runtime.InteropServices.DllImportAttribute.CharSet> field, defaults to ANSI. Some functions can have more than two versions.  
+ For example, specifying the **MessageBox** function in the User32.dll identifies the function (**MessageBox**) and its location (User32.dll, User32, or user32). The Microsoft Windows application programming interface (Windows API) can contain two versions of each function that handles characters and strings: a 1-byte character ANSI version and a 2-byte character Unicode version. When unspecified, the character set, represented by the <xref:System.Runtime.InteropServices.DllImportAttribute.CharSet> field, defaults to ANSI. Some functions can have more than two versions.  
```

Comment:
Yesterday, you'd asked whether this was discussed in the docs, and I didn't think that it was. Here at least is a rudimentary discussion.